packages/report-server/src/__tests__/reportBuilder/output/rawDataExport.test.ts

```diff
@@ -27,7 +27,8 @@ describe('rawDataExport', () => {
     });
   });
 
-  it('Add the rest of data elements that has no value into columns', async () => {
+  // legacy test from outputContext for raw data table output. Due to this being deprecated, now the output will behave exactly the same as rows and columns.
```

Comment:
What makes this test legacy?

Reply:
@IgorNadj this is legacy because previously with output context, even if not all data elements had values, all data elements from data group would become columns. Where as now, it doesn't care about data group or any extra context, the only columns that get printed are those literally in the report following transforms.

Reply:
Discussed, test covers how old behaviour changed to new